File: Data Structures & Algorithms/duplicate-integer/submission-1.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        int n = nums.length;

        Set<Integer> duplicate = new HashSet<>();

        for(int i = 0; i < n; i++){
            if(duplicate.contains(nums[i])){
                return true;
            }
            duplicate.add(nums[i]);
        }

        return false;
    }
}

File: Data Structures & Algorithms/remove-element/submission-1.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        start = 0
        n = len(nums)

        while(start < n):
            if nums[start] == val:
                nums[start] = nums[n - 1]
                n -= 1
            else:
                start += 1
        return n